File: Projects/server/Contracts/Responses/PersonDto.cs
namespace server.Contracts.Responses
{
    public class PersonDto
    {
        public uint PersonId { get; set; }
        public string Email { get; set; } = null!;
        public string Phone { get; set; } = null!;
        public string LastName { get; set; } = null!;
        public string FirstName { get; set; } = null!;
        public string? Patronymic { get; set; }
        public DateOnly Birth {  get; set; }
        public string? DriveLicense { get; set; }
        public byte RoleId { get; set; }
    }
}

File: Projects/DataProviderApp/app/src/main/java/com/example/dataproviderapp/dto/responses/PersonDto.kt
package com.example.dataproviderapp.dto.responses

import java.time.LocalDate

data class PersonDto(
    val personId: UInt,
    val email: String,
    val phone: String,
    val lastName: String,
    val firstName: String,
    val patronymic: String?,
    val birth: LocalDate,
    val driveLicense: String?,
    val roleId: UByte,
    val avatarId: UInt
)
